prover/src/provers/executor.rs

```diff
@@ -0,0 +1,65 @@
+use crate::contexts::SplitContext;
```

Comment:
it's best to move the executor out of prover folder, and put it into executor folder.